tls/extensions/s2n_extension_type.h

```diff
@@ -16,9 +16,10 @@
 #pragma once
 
 #include "stuffer/s2n_stuffer.h"
+#include "tls/s2n_tls_parameters.h"
 
-typedef uint8_t s2n_extension_type_id;
-extern const s2n_extension_type_id s2n_unsupported_extension;
+#define S2N_SUPPORTED_EXTENSIONS_COUNT          (sizeof(s2n_supported_extensions) / sizeof(s2n_supported_extensions[0]))
+#define S2N_SUPPORTED_EXTENSIONS_BITFIELD_LEN   ((S2N_SUPPORTED_EXTENSIONS_COUNT / sizeof(char)) + 1)
```

Comment:
it's not obvious to me what the +1 is for?

Reply:
Division ignores remainders :)